src/components/quote/code/views/quoteView.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+module.exports = React.createClass({
+
+  propTypes: {
+    purpose: React.PropTypes.oneOf(['default', 'primary', 'secondary', 'success', 'warning', 'danger', 'info']),
```

Comment:
Good idea - I have created an [issue](https://github.com/holidayextras/ui-toolkit/issues/39) for this